effectsize_example.Rmd

```diff
@@ -85,7 +100,7 @@ library(ggstance)   # for geom_pointrangeh()
 
 ```{r boilerplate, include = FALSE}
 set.seed(12)
-format_num <- function(...) format(..., digits = 2)
+format_num <- function(x, k = 2) format(round(x, k), nsmall = k)
```

Comment:
My second concern is that I tried to replace `signif` in Steve's `formatGES()`. The result is an inconsistency in the number of decimal places as shown below (replaced with `format_num(..., 3)`

> layout: F<sub>1.00,5.00</sub> = 415.82, η<sup>2</sup><sub>G</sub>
 = 0.46
> size: F<sub>2.00,**10.0**</sub> = 104.82, η<sup>2</sup><sub>G</sub>
 = 0.36
> layout × size: F<sub>2.00,**10.0**</sub> = 101.28, η<sup>2</sup><sub>G</sub>
 = 0.17
> color did not have a substantive effect (F<sub>3.00,**15.0**</sub> = **1.2686**, η<sup>2</sup><sub>G</sub>
 = 0.0035)